.gitignore

```diff
@@ -2,5 +2,9 @@ target/
 overlays/
 .gitignore
 .idea/
-*.iml
-**/*.prefs
+.settings/
```

Comment:
I believe this is already being done on KFS6. I know for certain that this should be done on Rice 2.3 for KC, and is being done on the KC repo for Eclipse, but not for the *.iml files.